Playground/GeneratePlayground.swift

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/swift
+import Foundation
+
+extension FileManager {
+    func directoryExists(at url: URL) -> Bool {
+        var isDirectory: ObjCBool = false
+        return fileExists(atPath: url.path, isDirectory: &isDirectory) && isDirectory.boolValue
+    }
+}
+
+let fileManager = FileManager()
+let isDebuggable: Bool = UserDefaults.standard.bool(forKey: "debug") // use -debug
+
+// Get the Playgrounds directory
+let playgroundsDirectory = URL(fileURLWithPath: fileManager.currentDirectoryPath)
+    .appendingPathComponent(CommandLine.arguments.first!)
```

Comment:
Let's use the subscript operator to remove the force unwrap.

```suggestion
    .appendingPathComponent(CommandLine.arguments[0])
```